anitya/templates/flags.html

```diff
@@ -0,0 +1,149 @@
+{% extends "master.html" %}
+
+{%block header %}
+{% if refresh %}
+<meta http-equiv="refresh" content="5" >
+{% endif %}
+{% endblock %}
+
+
+{% block body %}
+
+<div class="page-header">
+  <h1>Flags</h1>
+  {% if refresh %}
+  <p>This page should refresh automatically every 5 seconds</p>
+  <a href="{{ url_for('browse_flags') }}?page={{page}}">
+    <button type="submit" class="btn btn-danger" id="checknow">
+      <span class="glyphicon glyphicon-remove-sign"></span>
+      Remove auto-refresh
+    </button>
+  </a>
+  {% else %}
+  <a href="{{ url_for('browse_flags') }}?page={{page}}&refresh=True">
```

Comment:
You cna pass ``page=page, refresh=True`` within the ``url_for()`` method that will add them automatically and nicely